languages/go/xorm/main.go

```diff
@@ -213,57 +175,15 @@ func main() {
 	switch fn {
 	case "setupDev":
 		setupDev()
-	case "runExamples":
-		runExamples()
 	default:
 		fmt.Println("Unknown function:", fn)
 	}
 }
 
 func setupDev() {
-	// Recreate gotest db on each run
-	setupDb()
-
 	// Connect to go test directly and create table
 	createTable()
 
-	// Install EQL and add config
-	installEql()
-}
-
-func runExamples() {
```

Comment:
As noted in the description, these examples are duplicates of the e2e tests. Removing as there's no need for both, tests are the source of truth for working examples.